pkg/configuration/configuration.go

```diff
@@ -122,6 +122,12 @@ const (
 
 	// defaultToolchainStatusRefreshTime is the default refresh period for ToolchainStatus
 	defaultToolchainStatusRefreshTime = "5s"
+
+	// varDeactivationDomainsExcluded is a string of space-separated domains that should be excluded from automatic user deactivation
+	varDeactivationDomainsExcluded = "deactivation.domains.excluded"
+
+	// defaultDeactivationDomainsExcluded is the default set of domains that should be excluded from automatic user deactivation
+	defaultDeactivationDomainsExcluded = "@redhat.com"
```

Comment:
Let's not use any domains by default. I would rather deactivate everything by default.

Reply:
Done in https://github.com/codeready-toolchain/host-operator/commit/3d97e57209f2a8ff7f5039f53906370909e9c60a